apps/teams-test-app/e2e-test-data/meeting.json

```diff
@@ -19,11 +19,20 @@
     },
     {
       "title": "getMeetingDetails API Call - Success",
+      "version": ">2.21.0",
       "type": "callResponse",
       "boxSelector": "#box_getMeetingDetails",
-      "expectedAlertValue": "getMeetingDetails called",
+      "expectedAlertValue": "getMeetingDetails called with shouldGetVerboseDetails: undefined",
       "expectedTestAppValue": "{\"details\":{\"id\":\"testDetailsId\",\"scheduledStartTime\":\"testStartTime\",\"scheduledEndTime\":\"testEndTime\",\"joinUrl\":\"testJoinUrl\",\"title\":\"testTitle\",\"type\":\"Unknown\"},\"conversation\":{\"id\":\"testConversationId\"},\"organizer\":{\"id\":\"testOrganizerId\",\"tenantId\":\"testTenantId\"}}"
     },
```

Comment:
You are doing correct here but the failure is raised because in host sdk side, someone only mocked `getMeetinGDetails called` for you in alert dialog. Thus, you need to go there and raise PR first to make the change. That is what we have synced and made an agreement off line.

Another thing I would like to point it out is that, please keep the testing data for back-compat testing purpose. In this place, you directly change the data to be tested from all of versions to `>2.21.0`, which is to say, we miss the data to be tested in version `<= 2.21.0`. I would recommend you to copy the original data and give a version to it. An example is shown below:

 {
      "title": "getMeetingDetails API Call - Success",
      "version": "<=2.21.0",
      "type": "callResponse",
      "boxSelector": "#box_getMeetingDetails",
      "expectedAlertValue": "getMeetingDetails called",
      "expectedTestAppValue": "{\"details\":{\"id\":\"testDetailsId\",\"scheduledStartTime\":\"testStartTime\",\"scheduledEndTime\":\"testEndTime\",\"joinUrl\":\"testJoinUrl\",\"title\":\"testTitle\",\"type\":\"Unknown\"},\"conversation\":{\"id\":\"testConversationId\"},\"organizer\":{\"id\":\"testOrganizerId\",\"tenantId\":\"testTenantId\"}}"
    },
    
Please assume I also made the same comments for other testing cases. (If there is any left for getMeetingDetails API)